standard_forms/outbound-invoice-landscape-with-logo/latest/report-manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "is_custom": false,
-  "version": "2.6.3",
+  "version": "2.6.4",
```

Comment:
Not creating a new `latest` seeing as we're only adding some attributes, prev versions are still compatible